apps/data/priv/repo/migrations/20170827220442_add_opportunity_type_and_project_promoted_token.exs

```diff
@@ -0,0 +1,16 @@
+defmodule Data.Repo.Migrations.AddOpportunityTypeAndProjectPromotedToken do
+  use Ecto.Migration
+
+  def change do
+    alter table(:opportunities) do
+      add :type, :string, null: true
+    end
+
+    alter table(:projects) do
```

Comment:
+ `promoted` will be used to designate which projects are promoted via mails and tweets.
+ `api_token` is a simple solution to #33